Pre srv3 fixes

nugged opened the pull request.

nugged committed:
Set server timezone, if one provided
(new variable: koha_server_timezone)

nugged committed:
Add server domain to hosts file
(Apache requires to have that too)

nugged committed:
Idempotency fix for proper "plack enabled" handling

nugged committed:
Idempotency fix for proper "Create Koha instance" step

nugged committed:
Installation of fi-FI translation (if enabled in config, btw)

Also first pre-step is for itempotency: checking if it's already present

kohaputti commented on tasks/main.yml:
```diff
@@ -1,3 +1,8 @@
+- name: Set server timezone to {{ koha_server_timezone }}
+  timezone:
+    name: "{{ koha_server_timezone }}"
+  when: koha_server_timezone|default(False)|bool
```
Please add this to the default variables with something like UTC timezone.

kohaputti commented on tasks/main.yml:
```diff
@@ -63,6 +63,28 @@
+  when: koha_install_fi_translation and koha_installed_translations.changed
+  register: koha_lng_install
+  failed_when: 
+    # TODO: this is temporary workaround with stderr check below:
```
We should check if this can be fixed in this role, by the looks of it it is possible.

nugged replied:
I made with by adding env variables MEMCACHED_SERVERS=127.0.0.1:11211 MEMCACHED_NAMESPACE=KOHA before **koha-translate** command invocation so it not warns now and passes ansible well.

nugged committed:
Add default state to koha_server_timezone

nugged committed:
Set memcached environment to avoid warnings

When installing translations there were two warnings from
koha-translate command, and workaround for script to not fail.

Now with env set workaround not needed too.
